caas/kubernetes/provider/bootstrap.go

```diff
@@ -65,11 +65,13 @@ const (
 	mongoDBContainerName   = "mongodb"
 	apiServerContainerName = "api-server"
 
+	startupGraceTime = 300
```

Comment:
comment saying this is seconds
is 5 minutes enough? for slow storage provisioning etc? or slow install hooks that need to apt install all the things on a slow network

Reply:
5 minutes for bootstrap? I think thats heaps